backend/vehicle/helpers.py

```diff
@@ -69,17 +69,21 @@ def parse_excel_to_vehicle(excel_file):
             "chassis_condition": row["Chassis"],
             "body_condition": row["Body"],
         }
-
+        if len(vehicle_data.get("model_number")) > 10:
+            continue
```

Comment:
Why are we skipping when unicode is more than 10?

Reply:
whoops forgot to remove that line, fixing it rn

Reply:
I'll request review once everything's fixed